packages/contracts/contracts/registry/PluginRepoRegistry.sol

```diff
@@ -23,11 +23,14 @@ contract PluginRepoRegistry is InterfaceBasedRegistry {
     /// @param pluginRepo The address of the plugin repository.
     event PluginRepoRegistered(string name, address pluginRepo);
 
+    /// @notice Thrown if the plugin name doesn't match the regex `[0-9a-z\-]`
+    error InvalidPluginName(string name);
```

Comment:
We will be registering an ENS subdomain, not sure if such an event name is misleading

Reply:
It is an error message

Reply:
changed it to subdomain